File: src/net/eduard/permissions/command/PermissionsUserSetPrefixCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.Mine;
import net.eduard.api.lib.manager.CommandManager;
import net.eduard.api.lib.modules.FakePlayer;
import net.eduard.permissions.EduPermissions;
import net.eduard.permissions.manager.PermissionsManager;
import net.eduard.permissions.manager.PermissionsPlayer;

public class PermissionsUserSetPrefixCommand extends CommandManager {

	public PermissionsUserSetPrefixCommand() {
		super("setprefix", "definirprefixo");
		setUsage("/permissions user setprefix <player> <permission>");
		setDescription("Definir o prefixo do jogador");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

		PermissionsManager manager = EduPermissions.getInstance().getManager();
		if (args.length < 4) {
			sendUsage(sender);
		} else {
			String nome = args[2];
			String prefixo = Mine.toChatMessage(args[3]);
			PermissionsPlayer user = manager.getPlayer(new FakePlayer(nome));

//			PermissionsGroup group = manager.getGroup(nome);
			if (user != null) {
				user.setPrefix(prefixo);
				sender.sendMessage(EduPermissions.getInstance().message("player-set-prefix")
						.replace("$player", "" + nome).replace("$prefix", prefixo));
			} else {
				sender.sendMessage(
						EduPermissions.getInstance().message("player-not-exists").replace("$player", "" + nome));
			}
//			
		}

		return true;
	}

}

File: src/main/java/net/eduard/permissions/command/PermissionsUserSetPrefixCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.hybrid.PlayerUser
import net.eduard.api.lib.command.Sender
import lib.modules.Mine
import net.eduard.permissions.api.PermissionsAPI
import net.eduard.permissions.core.PermsMessages

class PermissionsUserSetPrefixCommand :
    Command("setprefix", "definirprefixo") {
    override fun onCommand(
        sender: Sender,
        args: List<String>
    ) {
        val manager = PermissionsAPI.instance
        if (args.size < 4) {
            sendUsage(sender)
            return
        }
        val nome = args[2]
        val prefixo = lib.modules.Mine.toChatMessage(args[3])
        val user = manager.getPlayer(PlayerUser(nome))

        user.suffix = prefixo
        user.update()
        sender.sendMessage(
            PermsMessages.message("player-set-prefix")
                .replace("\$player", "" + nome).replace("\$prefix", prefixo)
        )
    }

    init {
        usage = "/permissions user setprefix <player> <permission>"
        description = "Definir o prefixo do jogador"
    }
}